h/static/scripts/group-forms/components/test/CreateGroupForm-test.js

```diff
@@ -0,0 +1,23 @@
+import { mount } from 'enzyme';
+
+import CreateGroupForm from '../CreateGroupForm';
+
+describe('CreateGroupForm', () => {
+  const createWrapper = () => mount(<CreateGroupForm />);
+
+  [
+    { counter: 'name', limit: 25 },
+    { counter: 'description', limit: 250 },
+  ].forEach(({ counter, limit }) => {
+    describe(`${counter} character counter`, () => {
+      it(`initially renders 0/${limit}`, () => {
+        const wrapper = createWrapper();
+        const counterNode = wrapper.find(
```

Comment:
```suggestion
        const counterEl = wrapper.find(
```

We usually use either no suffix or "El" (for "DOM *el*ement").

Reply:
Done